apps/judicial-system/web/src/components/InfoCard/DefendantInfo/DefendantInfo.tsx

```diff
@@ -88,8 +88,9 @@ export const DefendantInfo: FC<DefendantInfoProps> = ({
         {displayDefenderInfo && (
           <Box display="flex" key={defendant.defenderName} role="paragraph">
             <Text as="span">{`${formatMessage(strings.defender)}: ${
-              defendant.defenderName ??
-              formatMessage(strings.noDefenderAssigned)
+              defendant.defenderName
+                ? defendant.defenderName
+                : formatMessage(strings.noDefenderAssigned)
```

Comment:
Ah, I should have spotted this in my earlier review. You could also do
```
defendant.defenderName || formatMessage(strings.noDefenderAssigned)
```
if you like that better.

Its sometimes a bit tricky to decide if you want to use `??` or `||`, the former is appropriate if you only want to replace `undefined` or `null` and the latter if you want to replace a falsy value. Most of the time I use `??` but that was obviously not correct in this situation.